libdnf/hy-query.h

```diff
@@ -32,8 +32,11 @@ G_BEGIN_DECLS
 #include "dnf-types.h"
 #include "hy-types.h"
 
-enum _hy_query_flags {
-    HY_IGNORE_EXCLUDES        = 1 << 0
+enum hy_exclude_flags {
+    HY_APPLY_EXCLUDES = 0,
+    HY_IGNORE_MODULAR_EXCLUDES = 1 << 0,
+    HY_IGNORE_NON_MODULAR_EXCLUDES = 1 << 1,
+    HY_IGNORE_EXCLUDES        = HY_IGNORE_MODULAR_EXCLUDES | HY_IGNORE_NON_MODULAR_EXCLUDES     
```

Comment:
This, IIUC, is a change in API: previously `HY_IGNORE_EXCLUDES = 1` and now `HY_IGNORE_MODULAR_EXCLUDES` and `HY_IGNORE_EXCLUDES = 3`. Can we change this? For `dnf` it will require both forward and backward compatibility version bumps.

Reply:
I changed design that allows 100% compatibility for C API.